go.mod

```diff
@@ -49,6 +50,7 @@ require (
 	github.com/pelletier/go-toml v1.9.3
 	github.com/prometheus/client_golang v1.12.1
 	github.com/sirupsen/logrus v1.8.1
+	github.com/smartystreets/goconvey v1.6.4
```

Comment:
What is this for? And what do they mean by `This software is FREE, AS IN PUPPIES, and is a gift. `
https://github.com/smartystreets/goconvey/blob/883c12515e6101b79f430987b4fd0ee50841bfe6/README.md

Reply:
remove this dependency.